File: inJava/app/src/main/java/com/barrygithub/rickandmortyapp/MyApp.java
package com.barrygithub.rickandmortyapp;

import android.app.Application;

import dagger.hilt.android.HiltAndroidApp;

/**
 * Project RickAndMortyApp
 * Created by Barry Zea H. on 3/04/23.
 * Copyright (c) Barry Zea H. All rights reserved.
 **/
@HiltAndroidApp
public class MyApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
    }
}

File: inKotlin/app/src/main/java/com/barrygithub/rickandmortyapp/MyApp.kt
package com.barrygithub.rickandmortyapp

import android.app.Application
import dagger.hilt.android.HiltAndroidApp

/**
 * Project RickAndMortyApp
 * Created by Barry Zea H. on 14/04/23.
 * Copyright (c) Barry Zea H. All rights reserved.
 *
 **/
@HiltAndroidApp
class MyApp : Application(){
    override fun onCreate() {
        super.onCreate()
    }
}